src/app/public/icons/icon-manifest.module.ts

```diff
@@ -2,13 +2,5 @@ import {
   NgModule
 } from '@angular/core';
 
-import {
-  SkyThemeIconManifestService
-} from './icon-manifest.service';
-
-@NgModule({
-  providers: [
-    SkyThemeIconManifestService
-  ]
-})
+@NgModule()
 export class SkyThemeIconManifestModule { }
```

Comment:
☝️ Do we need this module anymore?

Reply:
We should probably keep it in for backwards compatibility, but add a deprecated message above it saying it's no longer needed and can be removed.

Reply:
Done.